clients/wallet/src/stellar_wallet.rs

```diff
@@ -708,7 +708,8 @@ mod test {
 
 				// new wallet created, with the previous destination address acting as "SOURCE".
 				let temp_wallet =
-					wallet_with_secret_key_for_storage(storage_path, inactive_secret_key);
+					wallet_with_secret_key_for_storage(storage_path, inactive_secret_key)
+						.expect("should return an arc rwlock wallet");
```

Comment:
Nit: Let's maybe rename all of them to 
```
"should return a wallet instance"
```
because I don't need to know that it's `Arc` and `RwLock` in this case.